openmetadata-ui/src/main/resources/ui/src/constants/constants.ts

```diff
@@ -173,6 +173,8 @@ export const visibleFilters = [
   'servicename',
 ];
 
+export const CHART_WIDGET_DAYS_DURATION = 14;
```

Comment:
Let's have such things in separate const file so that it will be managable

Reply:
will be handling this in next PR